tests/ttnn/unit_tests/test_reshape.py

```diff
@@ -384,3 +384,39 @@ def test_fp32_support(input_shape, output_shape, device):
     output = ttnn.to_torch(ttnn_output)
 
     assert_with_pcc(torch_result, output, 0.9999)
+
+
+@pytest.mark.parametrize(
+    "input_shape, output_shape",
+    [
+        ([0], [0, 1]),
+        ([0], [1, 0]),  # failed
+        ([0, 5], [0, 0, 5]),
+        ([5, 0], [0, 5, 0]),  # failed
+    ],
+)
+@pytest.mark.parametrize(
+    "layout",
+    [ttnn.ROW_MAJOR_LAYOUT, ttnn.TILE_LAYOUT],
+)
+@pytest.mark.parametrize(
+    "ttnn_reshape",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "use_device, memory_config",
+    [(True, None), (True, ttnn.L1_MEMORY_CONFIG), (False, None)],
+)
```

Comment:
Do we need 48 different testing iterations for this case? I worry it increases test time too much

Reply:
I think it's all zero element won't spend too much time, or I'll reduce the cases

Reply:
Can you profile how long that component of the test takes in seconds by running it? Pytest has a lot of overhead

Reply:
The origin `test_reshape.py` spent `23.784s`, after added these 48 cases it spent `23.814s`, but I think I'll reduce it to about 8 case

Reply:
Oh then that's fine. FYI for the future you can use -k 'name of function' in pytest to run just that function and not the whole file.

Reply:
okok, thank you